products/composer.md

```diff
@@ -7,10 +7,10 @@ changelogTemplate: "https://getcomposer.org/changelog/__LATEST__"
 releases:
   - releaseCycle: "2.2"
     eol: 2023-12-31
-    release: 2021-12-22
-    latest: "2.2.7"
+    release: 2022-03-15
```

Comment:
Apparently I misunderstood the meaning of this value, because this is the release date of the _latest_ version in the branch, not _initial_. I'm reverting this - [version 2.2.0](https://getcomposer.org/changelog/2.2.0) was indeed released on 2021-12-22.